website/docs/docs/dbt-versions/release-notes/72-Feb-2024/job-completion-trigger.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "New: Trigger on job completion"
+description: "February 2024: Native support now available in dbt Cloud for triggering deploy jobs when other deploy jobs finish."
+sidebar_label: "New: Trigger on job completion"
+sidebar_position: 07
+tags: [Feb-2024]
+date: 2024-02-15
+---
+
+# New: Trigger on job completion <Lifecycle status="team,enterprise" />
+
+Now available for dbt Cloud Team and Enterprise plans is the ability to trigger deploy jobs when other deploy jobs complete. You can enable this feature [in the UI](/docs/deploy/deploy-jobs) with the  **Run when another job finishes** option or with the [Create Job API endpoint](/dbt-cloud/api-v2#/operations/Create%20Job). 
```

Comment:
If we list the UI option in the release note, I feel it might benefit the customers to take them the whole way there and tell them where it is. We do link to the page but the feature is a little buried in text there. 
```suggestion
Now available for dbt Cloud Team and Enterprise plans is the ability to trigger deploy jobs when other deploy jobs are complete. You can enable this feature [in the UI](/docs/deploy/deploy-jobs) with the  **Run when another job finishes** option in the **Triggers** section of your job or with the [Create Job API endpoint](/dbt-cloud/api-v2#/operations/Create%20Job). 
```

Reply:
great flag!